[2024-04-12T16:32:52.715+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-04-11T00:00:00+00:00 [queued]>
[2024-04-12T16:32:57.965+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-04-11T00:00:00+00:00 [queued]>
[2024-04-12T16:33:01.009+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-04-12T16:33:01.016+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-04-12T16:33:01.027+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-04-12T16:33:01.540+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2024-04-11 00:00:00+00:00
[2024-04-12T16:33:01.891+0000] {standard_task_runner.py:55} INFO - Started process 19898 to run task
[2024-04-12T16:33:02.593+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2024-04-11T00:00:00+00:00', '--job-id', '209', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpou7f2jab']
[2024-04-12T16:33:02.982+0000] {standard_task_runner.py:83} INFO - Job 209: Subtask create_table
[2024-04-12T16:33:08.154+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2024-04-11T00:00:00+00:00 [running]> on host a868c0c748e3
[2024-04-12T16:33:18.370+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2024-04-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-11T00:00:00+00:00
[2024-04-12T16:33:18.530+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-04-12T16:33:18.578+0000] {sql.py:315} INFO - Running statement: 
			CREATE TABLE IF NOT EXISTS users (
				firstname TEXT NOT NULL,
				lastname TEXT NOT NULL,
				country TEXT NOT NULL,
				username TEXT NOT NULL,
				password TEXT NOT NULL,
				email TEXT NOT NULL
			);
		, parameters: None
[2024-04-12T16:33:18.587+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2024-04-12T16:33:18.824+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20240411T000000, start_date=20240412T163254, end_date=20240412T163318
[2024-04-12T16:33:19.768+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-04-12T16:33:21.490+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
